python/cudf_polars/cudf_polars/dsl/ir.py

```diff
@@ -883,13 +883,11 @@ def __init__(
             raise NotImplementedError("dynamic group by")
         if any(GroupBy.check_agg(a.value) > 1 for a in self.agg_requests):
             raise NotImplementedError("Nested aggregations in groupby")
-        self.agg_infos = [req.collect_agg(depth=0) for req in self.agg_requests]
         self._non_child_args = (
             self.keys,
             self.agg_requests,
             maintain_order,
             options,
-            self.agg_infos,
```

Comment:
I also found that `options` can be an unserializable Polars objects (though the only thing we actually use in `do_evaluate` is `options.slice`).

Reply:
Right, do you see any tests we could rely immediately upon for this case?